screencaptor/src/main/java/com/wealthfront/screencaptor/ScreenCaptor.kt

```diff
@@ -102,25 +102,26 @@ object ScreenCaptor {
       Log.d(SCREENSHOT, "Disabling views: $viewIdsToExclude")
       val initialStateOfViews = ViewVisibilityModifier.hideViews(view, viewIdsToExclude)
       val initialDataOfViews = dataProcessor.modifyViews(view, viewModifiers)
-      view.requestLayout()
```

Comment:
Do we not need to explicitly request a layout anymore?

Reply:
Yup! This post just covers it.